### PR TITLE
gha: fix artefacts build on ppc64le

### PR DESCRIPTION
**Pull request by Amulyam24**
Add step in the right place to prepare the runner for the builds/tests.

Fixes: #8525

### CODE REVIEW

**Review by stevenhorsman — APPROVED** [review]
LGTM

File: .github/workflows/build-kata-static-tarball-ppc64le.yaml [hunk 1]
```diff
@@ -38,6 +38,9 @@ jobs:
       - name: Adjust a permission for repo
         run: |
           sudo chown -R $USER:$USER $GITHUB_WORKSPACE
+      
+      - name: Prepare the self-hosted runner
```
Comment (BbolroC) [inline .github/workflows/build-kata-static-tarball-ppc64le.yaml:42]: I would be better to place this step after the checkout, just opening up a room to make an urgent patch against the repo. What do you think?

**Review by BbolroC — APPROVED** [review]
Other than the suggestion for the location of the step, it looks good to me. Thanks @Amulyam24 !

File: .github/workflows/build-kata-static-tarball-ppc64le.yaml (hunk 1 above)
Comment (Amulyam24) [reply .github/workflows/build-kata-static-tarball-ppc64le.yaml:42]: > just opening up a room to make an urgent patch against the repo.

Hi @BbolroC, I don't mind moving it to after checkout but can I ask what you mean by this? I didn't get you completely

**Review by GabyCT — APPROVED** [review]

File: .github/workflows/build-kata-static-tarball-ppc64le.yaml (hunk 1 above)
Comment (BbolroC) [reply .github/workflows/build-kata-static-tarball-ppc64le.yaml:42]: Let's say what if you need to do something against the source code before the real steps start? (I know it is very unlikely to happen). I just wanted to inform you of this kind of situation in which I experienced before.

Comment (Amulyam24) [reply .github/workflows/build-kata-static-tarball-ppc64le.yaml]: Got it, makes sense. Changes applied, Thanks @BbolroC !

**Pull request merged**